portfolio/src/main/webapp/comments.html

```diff
@@ -24,6 +24,7 @@
                 <a class="nav-item nav-link" href="#about">About</a>
 
                 <a class="nav-item nav-link" href="#projects">Projects</a>
```

Comment:
I don't think these in-page links (`#projects`) will work from another page.. An extra slash or full path ought to fix it (`/#comments` or `/index.html#projects`).

(Also applies to the navbar on the maps page